packages/ceph/ceph.20240106/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "Ahrefs Pte Ltd <github@ahrefs.com>"
+authors: [ "Ahrefs Pte Ltd <github@ahrefs.com>" ]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ahrefs/ocaml-ceph"
+dev-repo: "git+https://github.com/ahrefs/ocaml-ceph.git"
+bug-reports: "https://github.com/ahrefs/ocaml-ceph/issues"
+tags: [ "org:ahrefs" "clib:stdc" "clib:cephfs"  ]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs "@install" "@doc" {with-doc}]
+]
+depends: [
+  "ocaml"
+  "dune" {>= "2.0"}
+  "odoc" {with-doc}
+  "ctypes"
```

Comment:
I'm not totally sure of what is going on with lower bounds, but it seems we need to add an extra lower bound somewhere:

https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/a0392dde580f0e846853e811cbf86b9767dc3df8/variant/compilers,4.03,ceph.20240106,lower-bounds


```
-> installed ctypes.0.4.0
-> installed dune.2.0.1
[ERROR] The compilation of ceph.20240106 failed at "dune build -p ceph -j 39 @install".

#=== ERROR while compiling ceph.20240106 ======================================#
# context              2.2.0~beta2~dev | linux/x86_64 | ocaml-base-compiler.4.03.0 | pinned(https://github.com/ahrefs/ocaml-ceph/releases/download/20240106/ceph-20240106.tbz)
# path                 ~/.opam/4.03/.opam-switch/build/ceph.20240106
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p ceph -j 39 @install
# exit-code            1
# env-file             ~/.opam/log/ceph-7-efad0e.env
# output-file          ~/.opam/log/ceph-7-efad0e.out
### output ###
#     ocamlopt src/.gen_structs_c.eobjs/native/dune__exe__Gen_structs_c.{cmx,o}
# File "_none_", line 1:
# Warning 58: no cmx file was found in path for module Cstubs_structs, and its interface was not compiled with -opaque
#       ocamlc src/gen_structs.o (exit 2)
# (cd _build/default/src && /home/opam/.opam/4.03/bin/ocamlc.opt -g -I /home/opam/.opam/4.03/lib/ctypes -ccopt -std=gnu99 -ccopt -O2 -ccopt -fno-strict-aliasing -ccopt -fwrapv -ccopt -fcommon -ccopt -Wall -ccopt -D_FILE_OFFSET_BITS=64 -ccopt -D_REENTRANT -ccopt -fPIC -ccopt -g -o gen_structs.o gen_structs.c)
# gen_structs.c: In function 'main':
# gen_structs.c:299:39: error: 'ENOENT' undeclared (first use in this function)
#   299 |      enum { check_ENOENT_const = (int)ENOENT };
#       |                                       ^~~~~~
# gen_structs.c:299:39: note: each undeclared identifier is reported only once for each function it appears in
```

Reply:
```suggestion
  "ctypes" {>= "0.8.0"}
```
This is maybe overkill, but seems to be enough on the first successful lower bound build